config/store.go

```diff
@@ -60,3 +61,47 @@ func NewStore(dsn string, watch bool) (Store, error) {
 
 	return NewFileStore(dsn, watch)
 }
+
+func MigrateConfig(sourceStore, destinationStore Store) error {
+	// Copy config from source to destination
+	sourceStoreConfig := sourceStore.Get()
+
+	_, err := destinationStore.Set(sourceStoreConfig)
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate config")
+	}
+
+	//get the certificate files from sourceStore
+	idpCertificateFile, err := sourceStore.GetFile(*sourceStoreConfig.SamlSettings.IdpCertificateFile)
```

Comment:
I believe this (and below) will fail if there is no configured IDP certificate -- let's check if these settings are empty before trying to migrate the files.